tutorials/extended_source_spectral_analysis.ipynb

```diff
@@ -0,0 +1,461 @@
+    "\n",
+    "In summary, we have to:\n",
+    "\n",
+    "- Define an ON region (a `~regions.SkyRegion`) fully enclosing the source we want to study.\n",
+    "- Define a geometry that fully contains the region and that covers the required energy range (beware in particular, the true energy range).  \n",
+    "- Create the necessary makers : \n",
+    "    - the map dataset maker : `~gammapy.cube.MapDatasetMaker`\n",
+    "    - the OFF background maker, here a `~gammapy.spectrum.ReflectedRegionsBackgroundMaker`\n",
+    "    - and usually the safe range maker : `~gammapy.cube.SafeRangeMaker`\n",
+    "- Perform the data reduction loop. And for every observation:\n",
+    "    - Produce a map dataset and squeeze it to a spectrum dataset with `~gammapy.cube.MapDataset.to_spectrum_dataset(on_region)`\n",
+    "    - Extract the OFF data to produce a `~gammapy.spectrum.SpectrumDatasetOnOff` and compute a safe range for it.\n",
+    "    - Stack or store the resulting spectrum dataset.\n",
+    "- Finally proceed with model fitting on the dataset as usual.\n",
+    "\n",
+    "Here, we will use the RX J1713-3945 observations from the H.E.S.S. first public test data release. The tutorial is implemented with the intermediate level API.\n",
+    "\n",
+    "## Setup \n",
+    "\n",
+    "As usual, we'll start with some general imports..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import astropy.units as u\n",
+    "from astropy.coordinates import SkyCoord, Angle\n",
+    "from regions import CircleSkyRegion\n",
+    "from gammapy.maps import Map, MapAxis, WcsGeom\n",
+    "from gammapy.modeling import Fit, Datasets\n",
+    "from gammapy.data import DataStore\n",
+    "from gammapy.modeling.models import (\n",
+    "    PowerLawSpectralModel,\n",
+    "    SkyModel,\n",
+    ")\n",
+    "from gammapy.cube import SafeMaskMaker, MapDataset, MapDatasetMaker\n",
+    "from gammapy.spectrum import ReflectedRegionsBackgroundMaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Select the data\n",
+    "\n",
+    "We first set the datastore and retrieve a few observations from our source."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datastore_path = \"/Users/terrier/Code/gammapy-dev/gammapy-datasets\"\n",
```

Comment:
The path should be modified to maybe $GAMMAPY_DATA

Reply:
Indeed. Thanks.